service/src/integrationTest/kotlin/fi/espoo/evaka/reservations/MissingHolidayReservationsRemindersTest.kt

```diff
@@ -178,13 +184,17 @@ class MissingHolidayReservationsRemindersTest : FullApplicationTest(resetDbBefor
                 )
             )
 
+            it.insert(DevEmployee(id = employeeId))
```

Comment:
we insert this employee here, but is it actually used anywhere?

Reply:
Unfortunately it needs to be in the database to satisfy the foreign key contraint when it is referenced from the foster_parent table.

Reply:
ah this employeeId is also used for admin. then the correct way is
```
    private val admin = DevEmployee(roles = setOf(UserRole.ADMIN))

    @BeforeEach
    fun beforeEach() {
        db.transaction { tx ->
            tx.insert(admin)
```